pkg/apis/fission.io/v1/const.go

```diff
@@ -96,3 +96,8 @@ const (
 	// set a max number for iterations to prevent infinite processing of canary config
 	MaxIterationsForCanaryConfig = 10
 )
+
+const (
+	// FissionDebugHeader is added to the http request to the function when debugging information is requested
+	FissionDebugHeader = "X-Fission-Debug"
```

Comment:
While the other comments can be addressed in later PRs, we'll have to remove this header before merging this PR -- this isn't something that the function's users should be able to set, since it exposes information about the function that the fission function authors may not want to expose to their users.